ui/src/components/Sidebar/AddGroupSidebarItem.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import * as Dropdown from '@radix-ui/react-dropdown-menu';
+import { useLocation, useNavigate } from 'react-router-dom';
+import AddIcon16 from '../icons/Add16Icon';
+import HomeIconMobileNav from '../icons/HomeIconMobileNav';
+import SidebarItem from './SidebarItem';
+
+export default function AddGroupSidebarItem() {
+  const navigate = useNavigate();
+  const location = useLocation();
+  const [open, setOpen] = useState(false);
+
+  const onOpenChange = (openChange: boolean) => {
+    setOpen(openChange);
+  };
+
+  const navigateToModal = (path: string) => {
+    navigate(path, { state: { backgroundLocation: location } });
+  };
+
+  return (
+    <Dropdown.Root open={open} onOpenChange={onOpenChange}>
+      <Dropdown.Trigger>
+        <SidebarItem
+          icon={<HomeIconMobileNav className="m-1 h-4 w-4" />}
+          actions={<AddIcon16 className="h-4 w-4" />}
+        >
+          <span data-testid="add-group-sidebar-button">Groups</span>
```

Comment:
Need some kind of indicator that we've been invited to a group because it's easily lost in the notifications on the right if you have a lot (also maybe that screen is going away eventually).

old:
<img width="1588" alt="groups" src="https://github.com/tloncorp/landscape-apps/assets/5466421/e55d5e3a-6a85-4eca-8ef5-621bb5b41f88">

new:
<img width="1588" alt="Screenshot 2024-01-23 at 5 29 58 PM" src="https://github.com/tloncorp/landscape-apps/assets/5466421/edc1c697-e27d-4268-ac6b-df72a11f9eef">

Reply:
Agreed, good thought. I think sticking em near the top of the sidebar like we do on mobile would make sense here?

Reply:
that would probably work yeah, we can address later if it gets too crazy

Reply:
Having their own section felt too heavy. Popped them into pinned since they're highly visible there and it's expected you'll act on them immediately.

<img width="1276" alt="invite" src="https://github.com/tloncorp/landscape-apps/assets/90741358/ac0a7d27-0e3b-4864-9a74-8824aa7c57c6">